core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -295,6 +296,12 @@ private[ui] class AllJobsPage(parent: JobsTab) extends WebUIPage("") {
                 <strong>Total Uptime: </strong>
                 {UIUtils.formatDuration(System.currentTimeMillis() - startTime)}
               </li>
+            } else {
```

Comment:
Hi Sean, yes, I can avoid the else statement. I updated the "Application Detail UI" for now. I will check the history server.